### DISCUSSION TITLE
2023/06/12/18.travel-to-hainan/

### DISCUSSION THREAD

**Discussion by giscus** [qllokirin/hexo_blog_comments#7] [Announcements]
# 2023/06/12/18.travel-to-hainan/

https://qllokirin.github.io/2023/06/12/18.travel-to-hainan/

**Comment by xgddyrq** [comment]
泰裤辣！！！！！！！！！！！！！！！！！！为什么没有吃海鲜

**Reply by qllokirin** [reply]
吃，现在在青岛刚吃了一顿 还挺好

**Comment by Zihan-W** [comment]
泰裤辣！！！！！！！！！！！！！！！！！！
